sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -368,8 +367,6 @@ private void performBasicMessagingCheck() {
     private Boolean setupDeviceAndProvider() {
         final var isConsumerEnabled =
                 injector.getInstance(Key.get(Boolean.class, Names.named(TestSuiteConfig.CONSUMER_ENABLE)));
-        final var isProviderEnabled =
-                injector.getInstance(Key.get(Boolean.class, Names.named(TestSuiteConfig.PROVIDER_ENABLE)));
 
         if (isConsumerEnabled) {
```

Comment:
"Enable" for the consumer is now a redundant option as well, can be removed from the config file and here